ui.apps/src/main/webpack.project/index.js

```diff
@@ -45,20 +45,7 @@ const WEBPACK = {
  * as strict as possible.
  */
 const ESLINT = {
-  // Optional: Replace `eslint:recommended` with `eslint-config-infield` and run
-  // `npm install --save-dev eslint-config-infield` for stricter linting rules
-  extends: "eslint:recommended",
```

Comment:
I purposefully placed those few lines into `webpack.project` so users of this Webpack setup have it easy to adjust the ESLint config in the right place. If this part gets moved into `webpack.core`, I suspect that people will modify the file in the `webpack.core` folder directly instead of making the effort of overriding the code via `webpack.project`.

Reply:
Yeah, I agree with you and I think you are correct. After I submitted this PR, I was looking into elegant ways to have a base config in webpack.core which is extended in webpack.project.

I also think it's rare for a company to need different ESLint rules for different projects. If that is needed, then a developer could create a new project-name/.eslintrc.js/json file which extends the default.

Thank you for reviewing this Kevin! Your feedback is really appreciated. This PR isn't done, and I should have marked it as a work in progress [WIP].